docs!: create docs and readme. (#217)
semiotic-ai/timeline-aggregation-protocol: README.md

## Process

1. **Opening a State Channel:** A state channel is opened via a blockchain contract, creating an EscrowAccount for the sender-receiver pair.
2. **Sending Receipts:** The sender sends receipts to the receiver through the state channel.
3. **Storing Receipts:** The receiver stores the receipts and tracks the aggregate payment.
4. **Creating a RAV Request:** A RAV request consists of a list of receipts and, optionally, the previous RAV.
5. **Signing the RAV:** The receiver sends the RAV request to the tap_aggregator, which signs it into a RAV.
6. **Tracking Aggregate Value:** The receiver tracks the aggregate value and new receipts since the last RAV.
7. **Requesting a New RAV:** The receiver sends new receipts and the last RAV to the tap_aggregator for a new RAV.
8. **Closing the State Channel:** When the allocation period ends, the receiver can send the last RAV to the blockchain and receive payment from the EscrowAccount.

## Performance Considerations

- The primary performance limitations are the time required to verify receipts and network limitations for sending requests to the tap_aggregator.

## Use Cases

> - The TAP protocol is suitable for systems with sequential operations that are too expensive to redeem individually on-chain. By aggregating operations and redeeming them in one transaction, costs can be reduced.

Reviewer: ```suggestion
- The TAP protocol is suitable for systems with sequential operations that are too expensive to redeem individually on-chain. By aggregating operations off-chain and redeeming them in one transaction, costs are drastically reduced.
```

## Process

1. **Opening a State Channel:** A state channel is opened via a blockchain 
contract, creating an EscrowAccount for the sender-receiver pair.
2. **Sending Receipts:** The sender sends receipts to the receiver through the 
state channel.
3. **Storing Receipts:** The receiver stores the receipts and tracks the 
aggregate payment.
4. **Creating a RAV Request:** A RAV request consists of a list of receipts and, 
optionally, the previous RAV.
5. **Signing the RAV:** The receiver sends the RAV request to the tap_aggregator, 
which signs it into a new RAV.
6. **Tracking Aggregate Value:** The receiver tracks the aggregate value and 
new receipts since the last RAV.
7. **Requesting a New RAV:** The receiver sends new receipts and the last RAV 
to the tap_aggregator for a new RAV.
8. **Closing the State Channel:** When the allocation period ends, the receiver 
can send the last RAV to the blockchain and receive payment from the EscrowAccount.

## Performance Considerations

- The primary performance limitations are the time required to verify receipts 
and network limitations for sending requests to the tap_aggregator.

## Use Cases

> - The TAP protocol is suitable for systems with sequential operations that are 

Reviewer: I think that'd be more accurate
```suggestion
- The TAP protocol is suitable for systems that need unidirectional, parallel micro-payments that are 
```
